lib/metrics/context.js

```diff
@@ -136,6 +138,21 @@ module.exports = function (log, config) {
       .then(() => data)
   }
 
+  function getToken (request) {
+    if (request.auth && request.auth.credentials) {
+      return request.auth.credentials
+    }
+
+    if (request.payload && request.payload.uid && request.payload.code) {
```

Comment:
This is what replaces the `fakeRequestToken` stuff that was removed from `lib/routes/account.js`.